sdk/src/main/java/com/easemob/im/server/api/chatgroups/GroupApi.java

```diff
@@ -5,24 +5,26 @@
 import com.easemob.im.server.api.chatgroups.admin.GroupAdminList;
 import com.easemob.im.server.api.chatgroups.admin.GroupAdminRemove;
 import com.easemob.im.server.api.chatgroups.announcement.GroupAnnouncement;
+import com.easemob.im.server.api.chatgroups.blacklist.add.GroupBlackListAdd;
```

Comment:
Please avoid using any offensive name. 
BTW, there are already block API located in com.easemob.im.server.api.block, please take a look.